Lab 01 - Data Engineering/01 - Delta Tables.py

```diff
@@ -421,7 +426,7 @@ def show_files_as_dataframe(files_location):
 # MAGIC delete from stores
 # MAGIC where id = 'BNE02';
 # MAGIC 
-# MAGIC SELECT * FROM table_changes('stores', 8, 9) -- Note that we increment versions due to UPDATE statements above
+# MAGIC SELECT * FROM table_changes('stores', 6) -- Note that we increment versions due to UPDATE statements above
```

Comment:
I tried running notebook with updated code and it fails on this line. If you run all cells in order, it would be version 8 and 9 that correspond to the updates done in this cell